### PR TITLE
Built dbt models and added cumulative metrics to models

### PR DESCRIPTION
**Pull request by Pratikshadange2005**

### CODE REVIEW

File: transform/models/marts/repo_pushes_daily.sql
Comment (edsioufi) [inline transform/models/marts/repo_pushes_daily.sql]: Small nit pick: I'd suggest using `order by date_day` to avoid repetition of the `date_trunc`.

Comment (edsioufi) [reply transform/models/marts/repo_pushes_daily.sql]: Similarly, you can do `sum(push_count)` to avoid repetition of `count(*)`.

File: transform/models/marts/repo_pushes_daily.sql
Comment (edsioufi) [inline transform/models/marts/repo_pushes_daily.sql]: Minor style: I prefer to do `group by 1, 2, 3` to avoid repetition of the column names. That's a subjective preference, I'll leave it up to you to decide :)

**Review by edsioufi — APPROVED** [review]
Awesome work on this! 🙌 

I've dropped some minor style improvement suggestions for your consideration, but I'm approving at once. If you decide to implement them, happy to re-review!

File: transform/models/marts/repo_pushes_daily.sql
Comment (Pratikshadange2005) [reply transform/models/marts/repo_pushes_daily.sql]: @edsioufi Updated SQL query; could you please review it?

File: transform/models/marts/repo_pushes_daily.sql
Comment (Pratikshadange2005) [reply transform/models/marts/repo_pushes_daily.sql]: @edsioufi Updated SQL query; could you please review it?

**Review by edsioufi — APPROVED** [review]
Looks perfect now! 🚀

**Pull request merged**
